runconfig/opts/parse.go

```diff
@@ -78,6 +78,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 		flCPUQuota          = cmd.Int64([]string{"-cpu-quota"}, 0, "Limit CPU CFS (Completely Fair Scheduler) quota")
 		flCpusetCpus        = cmd.String([]string{"-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems        = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flDiskQuota         = cmd.String([]string{"-disk-quota"}, "", "Limit disk quota (Now only support btrfs)")
```

Comment:
Was thinking the same, see https://github.com/docker/docker/pull/19651#issuecomment-174405726